test/parking_location/configs/parking_location_choice_annotate_trips_preprocessor.csv

```diff
@@ -0,0 +1,5 @@
+Description,Target,Expression
+Trip mode is drive,drive_trip,df.trip_mode <= 8
+Parking segment,parking_segment,setting('segment')
+Parking eligible trip,is_park_eligible,(drive_trip & (df.dest_purpose != "Home"))
+Activity duration,activity_duration,df.activity_duration_in_hours
```

Comment:
I don't see activity_duration_in_hours set anywhere?  To calculate trip duration, it is the depart time minus the next trip on the tour.

Reply:
we calculate `activity_duration_in_hours` for tours data as a pre-processing. Assuming this will be computed somewhere upstream in ASIM before it is needed in the parking location choice component.

Reply:
Ok, sounds good.